### ISSUE TITLE
Running `pip install ptvsd --pre` fails on latest version (4.1.1a6) on macOS and Linux.

### ISSUE THREAD

**Issue by d3r3kk** [Microsoft/ptvsd#492]
Latest version of ptvsd on pypi is causing failures in our vscode-python build, and fails for anyone who wishes to install the latest pre-release version.

OS: macOS, Linux (ubuntu 16.04, 18.04)
Python Version: 3.6

## Repro Steps: ##
1. Open shell prompt
2. Create a folder and cd into it
````
$ mkdir test
$ cd test
````
3. Install the latest ptvsd version from pypi.
````
$ python3 -m pip install ptvsd --pre --no-cache-dir
````
## Observed results ##
**Linux (local machine):**
````
dekeeler@dek-laptop-lnx:~/dev/github/d3r3kk/test$ python3 -m pip install ptvsd --pre --no-cache-dir
Collecting ptvsd
  Downloading https://files.pythonhosted.org/packages/1d/0c/6602d8651dd699fd86873bd0fbfc06fb2d006429cdd704f4aca84e71a819/ptvsd-4.1.1a6.zip (1.3MB)
    100% |████████████████████████████████| 1.3MB 25.7MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-s81f0btr/ptvsd/setup.py", line 42, in <module>
        with open('DESCRIPTION.md', 'r') as fh:
    FileNotFoundError: [Errno 2] No such file or directory: 'DESCRIPTION.md'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-s81f0btr/ptvsd/
````

**macOS (VSTS agent):**

````
2018-06-15T21:58:27.1509150Z Collecting ptvsd
2018-06-15T21:58:27.3701220Z   Downloading https://files.pythonhosted.org/packages/1d/0c/6602d8651dd699fd86873bd0fbfc06fb2d006429cdd704f4aca84e71a819/ptvsd-4.1.1a6.zip (1.3MB)
2018-06-15T21:58:28.1469500Z     Complete output from command python setup.py egg_info:
2018-06-15T21:58:28.1485150Z     Traceback (most recent call last):
2018-06-15T21:58:28.1500190Z       File "<string>", line 1, in <module>
2018-06-15T21:58:28.1517930Z       File "/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-install-nn9uwb_j/ptvsd/setup.py", line 42, in <module>
2018-06-15T21:58:28.1535890Z         with open('DESCRIPTION.md', 'r') as fh:
2018-06-15T21:58:28.1553570Z     FileNotFoundError: [Errno 2] No such file or directory: 'DESCRIPTION.md'
2018-06-15T21:58:28.1569250Z     
2018-06-15T21:58:28.1585540Z     ----------------------------------------
2018-06-15T21:58:28.1867370Z Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/pip-install-nn9uwb_j/ptvsd/
````

## Expected Result ## 
ptvsd is installed correctly.

**Comment by d3r3kk** [comment]
[See the `pip install ptvsd` task in this build.](https://vscode-python.visualstudio.com/VSCode-Python/VSCode-Python%20Team/_build/results?buildId=264&view=logs)

**Comment by d3r3kk** [comment]
Windows works fine with the lastest version, and the latest version works under Linux when pulling from master:

````
python3 -m pip install -t ./pythonFiles/experimental/ git+https://github.com/Microsoft/ptvsd/
````

**Comment by karthiknadig** [comment]
Will keep this open till a new release is made.

**Comment by karthiknadig** [comment]
#493 fixed the issue. Confirmed on linux
```
~$ python3 -m pip install --user ./ptvsd-4.1.1a6+1.gac33985.zip
Processing ./ptvsd-4.1.1a6+1.gac33985.zip
Building wheels for collected packages: ptvsd
  Running setup.py bdist_wheel for ptvsd ... done
  Stored in directory: /home/kanadig/.cache/pip/wheels/0e/41/ef/180053767381596cd58f3ea87fd1ba62e6a6abecb9c30fd411
Successfully built ptvsd
Installing collected packages: ptvsd
Successfully installed ptvsd-4.1.1a6+1.gac33985
```

**Comment by karthiknadig** [comment]
This issue should be fixed in v4.1.1a7.
